dspace-api/src/main/java/org/dspace/scripts/filepreview/FilePreview.java

```diff
@@ -58,8 +60,8 @@ public class FilePreview extends DSpaceRunnable<FilePreviewConfiguration> {
      */
     private String specificItemUUID = null;
 
-    private String email = null;
-    private String password = null;
+    private String epersonMail = null;
```

Comment:
Do we need this attributes?

Reply:
This attribute is for logging in from Cmder. Also as epersonPAssword.